pkgs/applications/science/networking/sumo/default.nix

```diff
@@ -0,0 +1,68 @@
+{ lib, bzip2, cmake, eigen, fetchFromGitHub, ffmpeg, fox_1_6, gdal,
+  git, gl2ps, gpp , gtest, jdk, libGL, libGLU, libX11, libjpeg,
+  libpng, libtiff, openscenegraph , proj, python3, python37Packages,
+  stdenv, swig, xercesc, xorg, zlib }:
+
+stdenv.mkDerivation rec {
+
+  pname = "sumo";
+  version = "1_8_0";
+
+  src = fetchFromGitHub {
+    owner = "eclipse";
+    repo = "sumo";
+    rev = "v${version}";
```

Comment:
```suggestion
    rev = "v${replaceStrings ["."] ["."] version}";
```

Reply:
Shouldn't it be `replaceStrings ["."] ["_"]`? Nice solution though

Reply:
yes